src/PrestaShopBundle/Resources/views/Admin/Multistore/header.html.twig

```diff
@@ -24,7 +24,13 @@
  *#}
 {% block multistore_header %}
   {% if isMultistoreUsed and app.request is not null %}
-    <div class="header-multishop{% if isAllShopContext %} header-multishop-allshops{% elseif currentContext.color is empty %} header-multishop-default{% endif %}{% if isTitleDark is same as (true) %} header-multishop-dark{% else %} header-multishop-bright{% endif %}">
+    <div
+        id="header-multishop"
+        class="header-multishop{% if isAllShopContext %} header-multishop-allshops{% elseif currentContext.color is empty %} header-multishop-default{% endif %}{% if isTitleDark is same as (true) %} header-multishop-dark{% else %} header-multishop-bright{% endif %}"
+        {% if isShopContext %}data-shop-id="{{ currentContext.id }}" {% elseif isGroupContext %}data-group-id="{{ currentContext.id }}" {% else %}data-allcontext="1" {% endif %}
```

Comment:
```suggestion
        {% if isShopContext %}data-shop-id="{{ currentContext.id }}" {% elseif isGroupContext %}data-group-id="{{ currentContext.id }}" {% else %}data-all-shops="1" {% endif %}
```

Reply:
Just to match the ShopContext const names
Also I'm not sure the pending space is needed in each if/elseif/else blocks

Reply:
Double fixed ;)